substrate/frame/support/src/traits/storage.rs

```diff
@@ -59,8 +59,33 @@ pub trait StorageInstance {
 	/// Prefix of a pallet to isolate it from other pallets.
 	fn pallet_prefix() -> &'static str;
 
+	/// Return the prefix hash of pallet instance.
+	/// NOTE: This hash must be `twox_128(pallet_prefix())`
+	/// Should not impl this function by hand. Only use the default or macro generated impls.
+	fn pallet_prefix_hash() -> [u8; 16] {
+		sp_io::hashing::twox_128(Self::pallet_prefix().as_bytes())
+	}
+
 	/// Prefix given to a storage to isolate from other storages in the pallet.
 	const STORAGE_PREFIX: &'static str;
+
+	/// Return the prefix hash of storage instance.
+	/// NOTE: This hash must be `twox_128(STORAGE_PREFIX)`.
+	fn storage_prefix_hash() -> [u8; 16] {
+		sp_io::hashing::twox_128(Self::pallet_prefix().as_bytes())
```

Comment:
Why is this also using the pallet prefix?

Reply:
Fixed